### PR TITLE
Custom Currency

### PR DESCRIPTION
**Pull request by gDanix**
Add support for a custom currency

This let the user define it's own currency and auto-save and auto-load (at start time) that configuration.

### CODE REVIEW

File: data/language/es-ES.txt [hunk 1]
```diff
@@ -4149,6 +4149,16 @@ STR_5877    :Software
 STR_5878    :Software (vía hardware)
 STR_5879    :OpenGL
 
+STR_5880	:Moneda personalizada
```
Comment (janisozaur) [inline data/language/es-ES.txt]: you should only supply `en-GB` with the change to main repo

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -4189,6 +4189,17 @@ STR_5877    :Software
 STR_5878    :Software (hardware display)
 STR_5879    :OpenGL
```
Comment (Nubbie) [inline data/language/en-GB.txt]: You can remove this spacing and I think you've added some extra indents (2 tabs instead of 1)

**Comment by Nubbie** [comment]
Don't think you should translate ``data/language/es-ES.txt`` in here as it might conflict with the merging bot
@Gymnasiast  Am I right?

**Comment by janisozaur** [comment]
@gDanix can you rebase this on top of current `develop` and get rid of merges?

**Comment by gDanix** [comment]
How can I do that?

It's the first time I use git that way, sorry

File: data/language/en-GB.txt (hunk 2 above)
Comment (Goddesen) [inline data/language/en-GB.txt]: This should be spaces, not tabs.

**Comment by janisozaur** [comment]
`git rebase -if origin/develop` where `origin` = this repo.

**Comment by gDanix** [comment]
Ok, right now I have to go, but later I'll try to fix all that merge s***

**Comment by Goddesen** [comment]
@gDanix Why did you close this..?

**Comment by gDanix** [comment]
It wan't intended. Github closed this PR because I resetted some commits.

I'll fix my commit history and create another PR.

**Comment by janisozaur** [comment]
you can still push to your `develop` and re-open… probably. that's why you shouldn't really work on `develop`, but have separate feature branches.

**Comment by gDanix** [comment]
I created a branch, but merged to develop before making this PR

**Comment by Gymnasiast** [comment]
Yes, but you open a pull request to pull from develop, not that branch. Ideally your develop branch should mirror upstream.

**Comment by gDanix** [comment]
So, the other PR (#3952) is OK?

**Pull request closed without merging**